analyses/cell-type-ewings/Dockerfile

```
@@ -1,9 +1,61 @@
-# A template docker file for creating a new analysis
-FROM ubuntu:22.04
+# Dockerfile for cell-type-ewings module
+FROM bioconductor/r-ver:devel-R-4.4.0
 
 # Labels following the Open Containers Initiative (OCI) recommendations
 # For more information, see https://specs.opencontainers.org/image-spec/annotations/?v=v1.0.1
+LABEL org.opencontainers.image.title="openscpca/cell-type-ewings"
+LABEL org.opencontainers.image.description="Docker image for the OpenScPCA analysis module 'cell-type-ewings'"
 LABEL org.opencontainers.image.authors="OpenScPCA scpca@ccdatalab.org"
-LABEL org.opencontainers.image.source="https://github.com/AlexsLemonade/OpenScPCA-analysis/tree/main/templates/analysis-module"
+LABEL org.opencontainers.image.source="https://github.com/AlexsLemonade/OpenScPCA-analysis/tree/main/analyses/cell-type-ewings"
 
+# Disable the renv cache to install packages directly into the R library
+ENV RENV_CONFIG_CACHE_ENABLED FALSE
+# set a name for the conda environment
+ARG ENV_NAME=openscpca-cell-type-ewings
 
+# set environment variables to install miniconda
+ENV PATH="/root/miniconda3/bin:${PATH}"
+ARG PATH="/root/miniconda3/bin:${PATH}"
```

I don't think we should need both of these: ENV sets environment variables within the final image, while ARG only sets them during the build, so having both is redundant. We want the former in this case, and I am going to suggest we follow the pattern of conda install from https://github.com/ContinuumIO/docker-images/blob/main/miniconda3/debian/Dockerfile, so change this to: 
```suggestion
ENV PATH="/opt/conda/bin:${PATH}"
```